### PR TITLE
Fix cni plugin for AWS scale tests

### PR DESCRIPTION
**Pull request by prateekgogia**

### CODE REVIEW

**Comment by hakman** [comment]
@prateekgogia @dims Amazon VPC CNI doesn't work with Ubuntu 22.04 (see https://github.com/aws/amazon-vpc-cni-k8s/issues/2103).

I suggest to also add the scale tests with Calico, as we know it's working.

**Comment by prateekgogia** [comment]
> @prateekgogia @dims Amazon VPC CNI doesn't work with Ubuntu 22.04 (see [aws/amazon-vpc-cni-k8s#2103](https://github.com/aws/amazon-vpc-cni-k8s/issues/2103)).
> 
> I suggest to also add the scale tests with Calico, as we know it's working.

Will create another follow up PR to change to Ubuntu 20.04, once this PR https://github.com/kubernetes/kops/pull/15557 passes the tests

**Comment by hakman** [comment]
/hold cancel
/lgtm cancel
```
config/jobs/kubernetes/kops/build_jobs.py:1068:0: C0301: Line too long (109/100) (line-too-long)
config/jobs/kubernetes/kops/build_jobs.py:1077:0: C0301: Line too long (109/100) (line-too-long)
```

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 1]
```diff
@@ -1065,18 +1065,26 @@ def generate_presubmits_scale():
...
             always_run=False,
+            env={
+                'CNI_PLUGIN': "amazonvpc",
+            }
         ),
         presubmit_test(
             name='presubmit-kops-aws-scale-amazonvpc-using-cl2',
             scenario='scalability',
+             # only helps with setting the right anotation test.kops.k8s.io/networking
             networking='amazonvpc',
```
Comment (ameukam) [inline config/jobs/kubernetes/kops/build_jobs.py:1079]:
```suggestion
            scenario='scalability',
            # only helps with setting the right anotation test.kops.k8s.io/networking
            networking='amazonvpc',
```

**Pull request merged**